src/modules/ie-objects/services/ie-objects.service.ts

```diff
@@ -343,28 +343,31 @@ export class IeObjectsService {
 	 * (not all details are in ES)
 	 */
 	public async findBySchemaIdentifier(
-		schemaIdentifier: string,
+		schemaIdentifier: string[],
 		referer: string,
 		ip: string
-	): Promise<IeObject> {
+	): Promise<IeObject[]> {
 		const { object_ie: objectIe } = await this.dataService.execute<
 			GetObjectDetailBySchemaIdentifierQuery,
 			GetObjectDetailBySchemaIdentifierQueryVariables
 		>(GetObjectDetailBySchemaIdentifierDocument, {
 			schemaIdentifier,
 		});
 
-		if (!objectIe[0]) {
-			throw new NotFoundException(`Object IE with id '${schemaIdentifier}' not found`);
-		}
-
-		const adapted = this.adaptFromDB(objectIe[0]);
-		adapted.thumbnailUrl = await this.playerTicketService.resolveThumbnailUrl(
-			adapted.thumbnailUrl,
-			referer,
-			ip
+		// if (!objectIe[0]) {
+		// 	throw new NotFoundException(`Object IE with id '${schemaIdentifier}' not found`);
+		// }
```

Comment:
move this to the wrapping function, so the 404 exception in the get single ie-object stays the same
```suggestion
```